sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -213,14 +212,21 @@ class DataFrameReader private[sql](sqlContext: SQLContext) extends Logging {
       url: String,
       table: String,
       parts: Array[Partition],
-      connectionProperties: Properties): DataFrame = {
+      connectionProperties: Properties,
```

Comment:
Thank you for reviewing the patch, Reynold. 
This particular jdbc method where I made the signature changes is not public. It i defined as   **private def jdbc** ..